### DISCUSSION TITLE
It is possible to hide the  menubottom of power

### DISCUSSION THREAD

**Discussion by ffs-akl** [lxqt/lxqt#2619] [Q&A] [answered]
Can I hide the menubutton in the following image by editing the lxqt-applications.menu?
<img width="324" alt="2024-09-22 143706" src="https://github.com/user-attachments/assets/0a4a8d53-6451-4f16-931d-f95c4b6f3ed4">

**Comment by stefonarch** [comment, marked as answer]
No, it isn't part of the menu file.

**Reply by ffs-akl** [reply]
Thank You！
I translated qps into Simplified Chinese (zh_CN), and today I found a couple of translation errors. But each time I can only get to the page shown in the screenshot below, and then I can't get to the next step.
Judging from this page, it should be relevant to you, and I don't know how to contact you, so here's something unrelated to the topic.
![swappy-20240922-153841](https://github.com/user-attachments/assets/ce15e85c-62d7-4bea-9ae7-21bdb08ddb36)

**Comment by stefonarch** [comment]
This is hosted.weblate which I tried maybe at the beginning, it's dead. Here you are:
https://translate.lxqt-project.org/projects/lxqt-desktop/qps/zh_CN/ search for the wrong strings

**Comment by ffs-akl** [comment]
I get this error information 
`Failure to complete registration. The e-mail address or username provided for this verification is already associated with another account.`

Is there a problem with my account? I always verify with my github account. No other accounts are linked

**Reply by stefonarch** [reply]
I don't think so, maybe it's a GH issue, try to log in directly (NOT register).

**Reply by ffs-akl** [reply]
I can log in directly and save in other project(eg. nm-tray),  but alwalys need log in again for lxqt project.

**Reply by ffs-akl** [reply]
Do I have to sign up for a new account?

**Reply by stefonarch** [reply]
nm-tray is on weblate org, our translations are on lxqt-project.org, that are 2 different instances.
https://translate.lxqt-project.org/
